backend/backend.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Do we want this file?

Reply:
Yes, I think so, I'm using visual studio, and if anyone else wants to to the same this file ties it all together

Reply:
Sure